javascripts/discourse/templates/components/dc-menu.hbs

```diff
@@ -14,6 +14,12 @@
         <a href="https://debtcollective.org/debt-union/" target="_blank" class="dc-header-link dc-menu-link">Join the Union</a>
         <a href="https://community.debtcollective.org/" class="dc-header-link dc-menu-link">Community</a>
         <a href="https://teespring.com/stores/debt-collective" target="_blank" class="dc-header-link dc-menu-link">Store</a>
+        <details class="dc-collapser">
+          <summary class="dc-header-link">Take Action!</summary>
+          <a href="https://community.debtcollective.org/calendar" class="dc-collapser-link" target="_blank">Events</a>
```

Comment:
Links within the community shouldn't be relative? like `/calendar`
and probably without the target blank